pkg/services/ngalert/eval/eval.go

```diff
@@ -86,7 +91,7 @@ const (
 )
 
 func (s State) String() string {
-	return [...]string{"Normal", "Alerting", "NoData", "Error"}[s]
+	return [...]string{"Normal", "Alerting", "Pending", "NoData", "Error"}[s]
```

Comment:
TIL this pattern 👍

Reply:
I learned about it when I added Pending, as well. It stopped me in my tracks for a second but made perfect sense once I let it sink in what was going on.